scripts/to_rst.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import re
+
+"""
+Assumption: 
+    1) Everything that starts with a comment is reStructuredText and
+       and the comment "#" is simply stripped.
+    2) Everything else is a codeblock and a "::" is prepended.
+    
+    To ensure 2), we use the variable "in_codeblock".
+"""
+
+def fix_inline_math(f):
+    """
+    In the code, I prefer to use inline math like        $\int \bm x \mathrm dx$,
+    so we have to convert it to rst inline math    :math:`\int \boldsymbol x \mathrm dx` 
+
+    This is done by using a regex to find groups of $something$ and 
+    replace the second one with `.
+    Afterwards, we replace the remaining $ with :math:`
+    """
+    f = re.sub(r"\\bm", r"\\boldsymbol", f)
+    f = re.sub(r"(\$[^\$]*)\$", r"\1`", f)
+    return re.sub(r"\$", r":math:`", f)
```

Comment:
This 10$ function will not work correctly for this 1$ sentence. But hey, it probably doesn't need to.